abci/versions.go

```diff
@@ -3,22 +3,39 @@ package abci
 import (
 	"errors"
 	"fmt"
-
 	"github.com/01builders/nova/appd"
+	"sort"
 )
 
 // ErrNoVersionFound is returned when no remote version is found for a given height.
 var ErrNoVersionFound = errors.New("no version found")
 
 // Version defines the configuration for remote apps.
 type Version struct {
+	Name        string
 	Appd        *appd.Appd
 	UntilHeight int64
 	PreHandler  string   // Command to run before starting the app
 	StartArgs   []string // Extra arguments to pass to the app
 }
 
-type Versions map[string]Version
+type Versions []Version
+
+// Sorted returns a sorted slice of Versions, sorted by UntilHeight (ascending).
+func (v Versions) Sorted() Versions {
```

Comment:
can retain the sorted function just so it doesn't matter what order they are entered